Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFDocumentLayout.mm

```diff
@@ -153,7 +157,8 @@
     currentYOffset -= pageMargin.height();
     currentYOffset += documentMargin.height();
 
-    m_documentBounds = FloatRect { 0, 0, std::max(maxPageWidth, availableWidth), currentYOffset };
+    m_scale = std::max<float>(availableWidth / maxPageWidth, 0.1);
```

Comment:
should the `0.1` here be extracted into a named `constexpr` value and then be used here for clarity? or a comment maybe?